### PR TITLE
Make submitting examples easier from the command line.

### PR DESCRIPTION
**Pull request by cckellogg**
New commands:
  heron examples list
  heron examples run <cluster> <example-id>

This patch is to provide easier commands to get the heron examples up and running.

### CODE REVIEW

File: heron/tools/common/src/python/utils/config.py [hunk 1]
```diff
@@ -95,6 +97,7 @@ def get_subparser(parser, command):
   for subparsers_action in subparsers_actions:
     # get all subparsers
     for choice, subparser in subparsers_action.choices.items():
+      print choice, subparser
```
Comment (nlu90) [inline heron/tools/common/src/python/utils/config.py]: is this `print` expected behavior?

**Comment by srkukarni** [comment]
This is a very interesting link between codes(java examples to python cli). Are we sure about this dep?

File: heron/tools/common/src/python/utils/config.py (hunk 1 above)
Comment (cckellogg) [reply heron/tools/common/src/python/utils/config.py]: no left over from debugging. will remove

**Comment by cckellogg** [comment]
It is a little different and strange but I think we need an easier way to get started with examples. Currently, the user needs to know where the examples are installed and the fully qualified class name. This is a lot to ask to get started. I'm open to suggestions on how make this better.

**Comment by huijunw** [comment]
With #2300, we have two sets of examples (heron-examples.jar and storm-compatibility-examples). Does this PR run the examples for both heron and storm-compatibility?

**Comment by srkukarni** [comment]
Closing this one

**Pull request closed without merging**